ui/page/privacy/setup_mixer_accounts_page.go

```diff
@@ -0,0 +1,304 @@
+													return layout.Flex{Alignment: layout.Middle}.Layout(gtx,
+														layout.Rigid(func(gtx C) D {
+															return layout.Inset{Bottom: values.MarginPadding12}.Layout(gtx, func(gtx C) D {
+																return ic.Layout(gtx, values.MarginPadding8)
+															})
+														}),
+														layout.Rigid(func(gtx C) D {
+															txt3 := `<span style="text-color: grayText2">
+													<b>Unmixed </b> account will be the change handling account.
+												</span>`
+
+															return layout.Inset{
+																Left: values.MarginPadding8,
+															}.Layout(gtx, renderers.RenderHTML(txt3, pg.Theme).Layout)
+														}),
+													)
+												})
+											}),
+										)
+									})
+								}),
+							)
+						}),
+						layout.Rigid(func(gtx C) D {
+							gtx.Constraints.Min.X = gtx.Constraints.Max.X
+							return pg.autoSetupClickable.Layout(gtx, pg.autoSetupLayout)
+						}),
+						layout.Rigid(func(gtx C) D {
+							gtx.Constraints.Min.X = gtx.Constraints.Max.X
+							return pg.manualSetupClickable.Layout(gtx, pg.manualSetupLayout)
+						}),
+					)
+				})
+			},
+		}
+		return page.Layout(gtx)
+	}
+
+	return components.UniformPadding(gtx, body)
+}
+
+func (pg *SetupMixerAccountsPage) autoSetupLayout(gtx C) D {
+	gtx.Constraints.Min.X = gtx.Constraints.Max.X
+	return layout.UniformInset(values.MarginPadding16).Layout(gtx, func(gtx C) D {
+		return layout.Flex{Spacing: layout.SpaceBetween}.Layout(gtx,
+			layout.Rigid(func(gtx C) D {
+				return layout.Flex{Alignment: layout.Middle}.Layout(gtx,
+					layout.Rigid(func(gtx C) D {
+						return pg.autoSetupIcon.Layout(gtx, values.MarginPadding20)
+					}),
+					layout.Rigid(func(gtx C) D {
+						autoSetupText := pg.Theme.H6("Auto setup")
+						txt := pg.Theme.Body2("Create and setup the needed accounts for you.")
+						return layout.Inset{
+							Left: values.MarginPadding16,
+						}.Layout(gtx, func(gtx C) D {
+							return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+								layout.Rigid(autoSetupText.Layout),
+								layout.Rigid(txt.Layout),
+							)
+						})
+					}),
+				)
+			}),
+			layout.Rigid(func(gtx C) D {
+				return layout.Inset{
+					Right: values.MarginPadding4,
+					Top:   values.MarginPadding10,
+				}.Layout(gtx, func(gtx C) D {
+					return pg.nextIcon.Layout(gtx, values.MarginPadding20)
+				})
+			}),
+		)
+	})
+}
+
+func (pg *SetupMixerAccountsPage) manualSetupLayout(gtx C) D {
+	gtx.Constraints.Min.X = gtx.Constraints.Max.X
+	return layout.UniformInset(values.MarginPadding16).Layout(gtx, func(gtx C) D {
+		return layout.Flex{Spacing: layout.SpaceBetween}.Layout(gtx,
+			layout.Rigid(func(gtx C) D {
+				return layout.Flex{Alignment: layout.Middle}.Layout(gtx,
+					layout.Rigid(pg.Icons.EditIcon.Layout24dp),
+					layout.Rigid(func(gtx C) D {
+						autoSetupText := pg.Theme.H6("Manual setup")
+						txt := pg.Theme.Body2("For wallets that have enabled privacy before.")
```

Comment:
Since this is not for only wallets that have enabled privacy before from our conversation, i believe this is misleading.
maybe  `Manually specify the wallet account to use for privacy set up.` or `Decide the wallet account to use for privacy set up.` will suffice.

Reply:
`For wallets that have enabled privacy before.` that feature is intended for that purpose, doesn't mean it can't be used even if privacy hasn't been enabled before.

That message is telling the user the use case for that option.

leaving as is.